feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle-dropdown/configurator-attribute-single-selection-bundle-dropdown.component.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { ChangeDetectionStrategy } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { NgSelectModule } from '@ng-select/ng-select';
+import { Configurator } from '../../../../core/model/configurator.model';
+import { ConfiguratorAttributeBaseComponent } from '../base/configurator-attribute-base.component';
+import { ConfiguratorAttributeSingleSelectionBundleDropdownComponent } from './configurator-attribute-single-selection-bundle-dropdown.component';
+import { ConfiguratorAttributeProductCardComponent } from '../../product-card/configurator-attribute-product-card.component';
+import { ConfiguratorShowMoreComponent } from '../../../show-more/configurator-show-more.component';
+import { I18nTestingModule } from '@spartacus/core';
+import { RouterTestingModule } from '@angular/router/testing';
+import { UrlTestingModule } from 'projects/core/src/routing/configurable-routes/url-translation/testing/url-testing.module';
+
+describe('ConfiguratorAttributeSingleSelectionBundleDropdownComponent', () => {
```

Comment:
these names are really long. we should consider changing the name convention here

Reply:
it is true, but name convention is similar in all form types components, so i thin we should rename it in other refactor task